teamscale_client/client.py

```diff
@@ -268,90 +350,97 @@ def upload_coverage_data(self, coverage_files, coverage_format, timestamp, messa
         """
         return self.upload_report(coverage_files, coverage_format, timestamp, message, partition)
 
-    def upload_report(self, report_files, report_format, timestamp, message, partition, move_to_last_commit=True):
+    def upload_report(
+            self, report_files: List[str], report_format: Union[ReportFormats, CoverageFormats],
```

Comment:
Might also be unit test results (as they are called in the client), right?

Reply:
Indeed 👍️